src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
```

Comment:
The last item in a block or list should not have a trailing comma or any other separator after it according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). The 'margin-right' should be the last property in the '.stars__star' selector, so it should not have a semicolon after it.